### ISSUE TITLE
Discussion: deprecation of brand icons

### ISSUE THREAD

**Issue by Banandrew** [feathericons/feather#763]
You may find this information useful, and I also feel like this decision may alleviate some pressure from the community. [Templarian/MaterialDesign](https://github.com/Templarian/MaterialDesign)—the community-driven extension for Google’s Material icons—came to a decision roughly a year ago to deprecate all brand icons. Here’s why ([source](https://dev.materialdesignicons.com/roadmap/brand-icons)):

> [...] brand and logo icons are simply not Material. Additionally, we have had issues dealing with logos that don't work well in a single color, don't fit in a distinguishable way within a 24x24dp artboard, or are just very obscure logos to begin with. It is very easy to say that we do not feel that we have served this niche of icons well.

Needless to say, it’s the same pattern for Feather: there’s just no way to include all requested logos and maintain the same level of quality for brand icons drawn in the style of Feather; logos have their own style (well, duh), and perhaps, there is no need to mix the two for the sake of aesthetic consistency.

[simple-icons/simple-icons](https://github.com/simple-icons/simple-icons) repository already covers everything, uses SVG and the same 24x24 `viewBox`, so it could be a great replacement for people that need brand icons.

What do you think? Please respond with thumbs up/down to this post and comment if you have something else to add.

**Comment by colebemis** [comment]
I totally agree with this. I wrote something similar in a recent [tweet](https://twitter.com/colebemis/status/1166503843292569600):
> We have GitHub issues for all those icons but don't expect them anytime soon. Logos are really tough to make with Feather's constraints. Personally, I'd recommend using something like Simple Icons (https://simpleicons.org) for brand icons.

I don't know if it's necessary to deprecate all the current logo icons. But I think this is good justification for not adding more.

**Comment by locness3** [comment]
Well, maybe add the ones awaiting merge ? @colebemis

**Comment by davewhitley** [comment]
At Automattic we have a separate repo for logos, called "[social logos](https://github.com/Automattic/social-logos)". You might consider making a separate repo for just logos, in the feather icon style.

The big difference with the Social Logos repo is that we respect the logo guidelines as much as possible. We don't add any style, but it's helpful to have a standard set all fit within the 24px grid.

**Comment by locness3** [comment]
Well, simpleicons fit in a 24px grid, and don't add any style, so not sure if a Feather version is needed if we both have yours and simpleicons

**Comment by leandrojo** [comment]
I disagree. I believe that social networks already have a minimalist logo in the creative process. Depreciating "social icons" deepens the problem of library coverage.

**Comment by PeterShaggyNoble** [comment]
Been meaning to post this for a while but we'd be more than happy to take on all your brand icon requests over at @simple-icons, as we've already done for Material Design Icons. Also being a member of the MDI team, I can say has been a huge help being able to offload all such requests to a dedicated library as they can become a time-sink, sourcing and designing them all, with diminishing returns the less popular a brand is. And then there's the difficult choice of whether or not you compromise your own design guidelines of violate a brand's guidelines when adding each one.

If a member of your team could take a run through all your issues & PRs, whether in progress, rejected or resolved, and slap a label on them to identify them, we'll begin working on fulfilling all requests we don't yet have in our library.

Going forward, there a are few ways we could handle any future requests. Firstly, you could add a note to your issue template outlining that you will not be adding any more brand icons and directing people to request them from us instead (see the [MDI template](https://github.com/Templarian/MaterialDesign/blob/master/.github/ISSUE_TEMPLATE/1_icon_request.md) for reference. For any requests that do still come in despite that, you could then:
- just reiterate what the template says and ask them to request it from us instead - in practice, I've noticed a very low uptake with this approach over at MDI,
- repost the request to our repo yourself, or,
- tag me or one of the other team members in a comment to bring it to our attention.

**Comment by mwskwong** [comment]
I personally disagree. You see, feather icons have their own style and aesthetics. If users absolutely want the original, then `simple-icons` is the way to go. Yet in certain cases, users are looking for icons that are visually recognizable but have the same aesthetics as the rest of the icons.

Take GitHub icon as an example, the one feather currently offers clearly has a different design from the original, but it looks more minimal and clearer when used together with other feather icons.

I believe we can do the same for other brand logos as well.

**Comment by jagirbahesh** [comment]
Icon Request: WordPress

**Comment by locness3** [comment]
> Icon Request: WordPress

I think you missed the point.

**Comment by LizAinslie** [comment]
This is sad :C

Are there any good community forks I could check out that prioritize brand marks?

Feather Icons look extremely good on my site, as pictured below, but as also pictured below there are icons I've had to utilize that don't match well :/
![screenshot of my personal site](https://i-work-at-the.cocaine.institute/Lizzy633a329cLOsEsvsKiGTL.png)

**Comment by karsa-mistmere** [comment]
@LizAinslie: Please note that it in fact goes against [Discord's brand guidelines](https://discord.com/branding) to modify its logo, as already discussed in #1156.

Brand icons are deprecated mostly for this very reason, not just in Feather but the [Lucide community fork](https://lucide.dev) as well.

**Comment by LizAinslie** [comment]
Wow! I hadn't expected to receive a reply this quickly. I presumed this thread was probably stale.

> @LizAinslie: Please note that it in fact goes against [Discord's brand guidelines](https://discord.com/branding) to modify its logo, as already discussed in #1156.
> 
> Brand icons are deprecated mostly for this very reason, not just in Feather but the [Lucide community fork](https://lucide.dev) as well.

@karsa-mistmere With all due respect that doesn't answer my question. I'm aware of Discord's brand guidelines and honestly I don't care that much; they haven't ever been too intent on even enforcing most parts of their ToS over the years so I doubt they'd care. I asked if anyone knew of an icon set that *did* modify the Discord logo similar to the style guide of Feather, not if it was allowed to modify it. I respect the decision to deprecate Brand Icons for this reason, but I have my own priorities. I'll probably just open up my vector software later and make one myself.

**Comment by karsa-mistmere** [comment]
> @karsa-mistmere With all due respect that doesn't answer my question. I'm aware of Discord's brand guidelines and honestly I don't care that much; they haven't ever been too intent on even enforcing most parts of their ToS over the years so I doubt they'd care. I asked if anyone knew of an icon set that _did_ modify the Discord logo similar to the style guide of Feather, not if it was allowed to modify it. I respect the decision to deprecate Brand Icons for this reason, but I have my own priorities. I'll probably just open up my vector software later and make one myself.

Please do actually look at #1156 before throwing a tantrum.

**Comment by karsa-mistmere** [comment]
@LizAinslie: There's already a Feather style Discord icon at #1156 (which I have linked for your convenience).

I don't believe you will find any stroke-only icon set that features a larger number of brand icons, for the very reasons Feather and Lucide have decided to deprecate them: not only do these violate most brand guidelines (which is probably fine on a personal homepage, but not really in even an open source project), they are oftentimes plain impossible to implement while also being restricted to the icon set's own guidelines.

If you really do need Feather style brand icons, you're best off fishing for them in icon request issues under https://github.com/feathericons/feather and https://github.com/lucide-icons/lucide or designing them yourself.

**Comment by LizAinslie** [comment]
All done! Thanks 👍🏻

![Screenshot](https://i-work-at-the.cocaine.institute/Lizzy633ab3aeHldaTUuXOLG7.png)

**Comment by jimmyff** [comment]
I understand that you need to draw the line somewhere so perhaps you could consider including brand icons that reach a certain level of global popularity? This could be something along the lines of 100 million monthly active users (MAU), this would limit the brands to a total of around 30 icons which would include popular requests such as whatsapp, tiktok, telegram etc -[statista publish this information annually](https://www.statista.com/statistics/272014/global-social-networks-ranked-by-number-of-users/).

Many people and websites, blogs and themes use these icons for social media links their headers and footers and it will become obsolete for this purpose without adding popular platform brand icons.

**Comment by DesignSkeptic** [comment]
Maybe merge simpleicons into this project so users only need to use 1 package. Similar to how bootstrap deals with their icons

**Comment by vincedubay-rei** [comment]
I agree but also, how active is this project? It has a ton of downloads on NPM but I'm seeing a lot of stale PRs and issues. I feel like more folks are going to be downloading this now that FontAwesome is charging for their CDN after you hit 10K views and what was formerly their free set of icons.

**Comment by awgv** [comment]
@vincedubay-rei [lucide-icons/lucide](https://github.com/lucide-icons/lucide) is a fork of Feather If you need an expanded icon set or ecosystem-specific packages.

**Comment by vincedubay-rei** [comment]
ooo yeah that seems really useful for my current project, thanks!
